src/actions.mli

```diff
@@ -0,0 +1,88 @@
+val job_action :
+     bot_info:Bot_components.Bot_info.t
+  -> Bot_components.GitLab_types.job_info
+  -> github_of_gitlab:(string -> string option)
+  -> unit Lwt.t
+
+val pipeline_action :
+     bot_info:Bot_components.Bot_info.t
+  -> Bot_components.GitLab_types.pipeline_info
+  -> github_of_gitlab:(string -> string option)
+  -> unit Lwt.t
+
+val coq_bug_minimizer_results_action :
+     bot_info:Bot_components.Bot_info.t
+  -> string
+  -> (Cohttp.Response.t * Cohttp_lwt__Body.t) Lwt.t
+
+val merge_pull_request :
+     bot_info:Bot_components.Bot_info.t
+  -> comment_info:Bot_components.GitHub_types.comment_info
+  -> unit Lwt.t
+
+val pull_request_updated :
+     bot_info:Bot_components.Bot_info.t
+  -> Bot_components.GitHub_types.issue_info
+     Bot_components.GitHub_types.pull_request_info
+  -> unit
+  -> gitlab_mapping:(string, string) Base.Hashtbl.t
+  -> github_mapping:(string, string) Base.Hashtbl.t
+  -> gitlab_of_github:(string -> string option)
+  -> (unit, string) Lwt_result.t
+
+val run_ci :
+     bot_info:Bot_components.Bot_info.t
+  -> comment_info:Bot_components.GitHub_types.comment_info
+  -> gitlab_mapping:(string, string) Base.Hashtbl.t
+  -> github_mapping:(string, string) Base.Hashtbl.t
+  -> gitlab_of_github:(string -> string option)
+  -> signed:bool
+  -> (Cohttp.Response.t * Cohttp_lwt__Body.t) Lwt.t
+
+val pull_request_closed :
```

Comment:
For consistency, this one should probably be renamed `pull_request_closed_action`.